internal/engine/analytics_reporter.go

```diff
@@ -71,9 +72,17 @@ func (ar *AnalyticsReporter) report() {
 				break
 			}
 			if !it.AnyLiveUpdateInfo().Empty() {
-				liveUpdateCount++
+				if !seenLU {
+					seenLU = true
+					liveUpdateCount++
```

Comment:
I just changed this stat to report the number of LiveUpdate _resources_ (rather than the number of LiveUpdate _targets_, which is now accounted for by the `multiImgLiveUpdate` value). This seems right to me but lmk if you disagree.